tests/kernel/poll/testcase.yaml

```diff
@@ -2,3 +2,4 @@ tests:
   kernel.poll:
     tags: kernel userspace
     min_ram: 16
+    platform_exclude: nrf52810_pca10040
```

Comment:
Could you extract the two platform_exclude changes into its own commit? And explain in the commit message why this platform is excluded.

Reply:
nrf52810_pca10040 platform will fail at these two test cases as issue #15103 described, and CI will run these two cases on that platform when this PR is submitted.

Reply:
A PR can contain multiple commits and the CI will check against the whole PR (including all commits).

Reply:
thanks,